x/claim/keeper/keeper.go

```diff
@@ -133,3 +133,24 @@ func (k Keeper) GetClaimRecords(ctx sdk.Context) []types.ClaimRecordAddress {
 
 	return claimRecords
 }
+
+// CreateModuleAccount set balance of airdrop module
+func (k Keeper) CreateModuleAccount(ctx sdk.Context, amount sdk.Coin) {
```

Comment:
I think this is not used?

Reply:
it's used in: claim_test.go#SetupClaimTest but let me try and move the logic over there

Reply:
removed